cloudferrylib/os/actions/utils.py

```diff
@@ -12,12 +12,13 @@
 # See the License for the specific language governing permissions and#
 # limitations under the License.
 
-from cloudferrylib.utils import utils
```

Comment:
Would be better to remove `from cloudferrylib.utils import utils as url` instead, saving 2 bytes with `url` is not worth it.